### ISSUE TITLE
outfit rules can not use Regex matching.

### ISSUE THREAD

**Issue by Fraktyl** [arkayenro/arkinventory#838]
If you use the Outfit() rule you have to match exact names.

If you change line 772 from:
if o and o.Name and string.lower( string.trim( o.Name ) ) == string.lower( string.trim( arg ) ) then

to
if o and o.Name and string.find(string.lower(o.Name), string.lower(string.trim(arg))) then

It will allow regex matching so you can match on things like:
outfit("DPS: %w+")

**Comment by arkayenro** [comment]
if i do that id have to remove the string.lower part to ensure the regex codes arent altered, not a huge deal though, just makes it case sensitive

**Comment by Fraktyl** [comment]
True, it would make it case sensitive.  I would love to have regex's for most of my rules.  Makes it easier to maintain them instead of things like:
outfit("Enhancement") or outfit("Affliction") and then having to edit the rule if I make a new spec.

Just my .02 and if you don't add it, it's not like I can't make the change myself to fill my personal needs.  Really appreciate the work done here.  My goto bag mod with WoW and I've converted a lot of folks to it once they see my layouts.

**Comment by arkayenro** [comment]
i'll look at how easy it is to an outfitregex( ) function by just modifying the existing function

if you dont care too much you know you can just use outfit() to pick up items in any outfit - its all ive really used because im not fussed which item is for which outfit, i just need to make sure i dont sell them when im not wearing them

**Comment by blueagewizard** [comment]
The problem I have with outfit() is when you have gear with the same name drop the rule adds it to that bar.
